src/components/Score.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+class Score extends Component {
+    render() {
+        return (
+            <div id='score'>The score is: {this.props.score >= 80 ? <span className='highScore'>{this.props.score}</span> :
```

Comment:
`<div id='score'>The score is: {score >= 80 ? <span className='highScore'>{score}</span>`

no more this.props if you define it at the top